trueusd/adapter.js

```diff
@@ -5,19 +5,28 @@ const customError = (data) => {
   return false
 }
 
+const convert = {
+  USD: 'totalTrust',
+  TUSD: 'totalToken',
+}
+
+const supplyParams = {
+  base: ['base', 'from', 'asset'],
+}
```

Comment:
I think instead of this we can just supply a param for "field" and parse that from responseData.

Reply:
My idea was to do this to normalize params in case we start supporting a lot of 'supply' feeds, but yeah this might be cleaner just as `field`.

Added a commit to change it.

Reply:
Ah, then maybe I like where you were going. So if `totalTrust` is the amount of USD secured, and `totalToken` are the number of tokens minted, then we could think about input parameters that would be compatible with multiple PoR-type feeds with different assets.

Reply:
How about an `endpoint` param where possible values are `secured` and `supply`? Secured maps to `totalTrust` and `supply` maps to `totalToken`. Trying to think if that would be compatible with future APIs for this same use-case.

Reply:
I like these endpoints suggestions.

Another slightly unrelated thing to think about is that Trust Token has stable coins for other fiat currencies (e.g. TrueCAD) that can be queried by changing the end of the url string. This adapter may be better as `trust-token` that takes in a param for `base` to switch between TUSD, TCAD, TAUD, ect.

Thoughts?

Reply:
If that is the case then definitely rename it as `trusttoken` and extend it to support different `endpoint`s  like `TrueUSD`, `TrueAUD`, etc.